engines/ceg_prophecy_arrays.cpp

```diff
@@ -93,6 +93,38 @@ CegProphecyArrays::CegProphecyArrays(const PonoOptions & opt,
 {
 }
 
+ProverResult CegProphecyArrays::prove()
```

Comment:
why is it in this PR?

Reply:
No longer changed after merging in master.